kafka-ui-react-app/src/components/Topics/Topic/Messages/MessagesTable.tsx

```diff
@@ -77,18 +77,19 @@ const MessagesTable: React.FC = () => {
               onPreview={() => setPreviewFor('content')}
             />
             <TableHeaderCell> </TableHeaderCell>
-
-            {previewFor !== null && (
-              <PreviewModal
-                values={previewFor === 'key' ? keyFilters : contentFilters}
-                toggleIsOpen={() => setPreviewFor(null)}
-                setFilters={(payload: PreviewFilter[]) =>
-                  previewFor === 'key'
-                    ? setKeyFilters(payload)
-                    : setContentFilters(payload)
-                }
-              />
-            )}
```

Comment:
@David-DB88 That's supposed to break the table. You can't just add a cell to a table without changing whole table html structure